test/dlc_tests/container_tests/bin/test_mx_dlc_telemetry_test.py

```diff
@@ -0,0 +1,81 @@
+import os
+import numpy as np
+import time
+
+def opt_in_opt_out_test():
+  os.environ['TEST_MODE']='1'
+  if os.path.exists("/tmp/test_request.txt"):
+    os.system("rm /tmp/test_request.txt")
+  os.environ["OPT_OUT_TRACKING"]="True"
+  cmd = "python -c 'import mxnet'"
+  os.system(cmd)
+  time.sleep(1)
+  assert(os.path.exists("/tmp/test_request.txt")==False)
+
+  if os.path.exists("/tmp/test_request.txt"):
+    os.system("rm /tmp/test_request.txt")
+  os.environ["OPT_OUT_TRACKING"]="False"
+  cmd = "python -c 'import mxnet'"
+  os.system(cmd)
+  time.sleep(1)
+  assert(os.path.exists("/tmp/test_request.txt")==True)
+
+
+  if os.path.exists("/tmp/test_request.txt"):
+    os.system("rm /tmp/test_request.txt")
+  os.environ["OPT_OUT_TRACKING"]="TRUE"
+  cmd = "python -c 'import mxnet'"
+  os.system(cmd)
+  time.sleep(1)
+  assert(os.path.exists("/tmp/test_request.txt")==False)
+
+  if os.path.exists("/tmp/test_request.txt"):
+    os.system("rm /tmp/test_request.txt")
+  os.environ["OPT_OUT_TRACKING"]="true"
+  cmd = "python -c 'import mxnet'"
+  os.system(cmd)
+  time.sleep(1)
+  assert(os.path.exists("/tmp/test_request.txt")==False)
+
+
+  if os.path.exists("/tmp/test_request.txt"):
+    os.system("rm /tmp/test_request.txt")
+  os.environ["OPT_OUT_TRACKING"]="XYgg"
+  cmd = "python -c 'import mxnet'"
+  os.system(cmd)
+  time.sleep(1)
+  assert(os.path.exists("/tmp/test_request.txt")==True)
+
+
+  print("Opt-In/Opt-Out Test passed")
+
+def performance_test():
+  os.environ['TEST_MODE']='0'
+  os.environ["OPT_OUT_TRACKING"]="False"
+  NUM_ITERATIONS = 5
+
+  for itr in range(NUM_ITERATIONS):
+    total_time_in = 0
+    for x in range(NUM_ITERATIONS):
+      cmd = "python -c 'import time; start= time.time();import mxnet; '"
+      start = time.time()
+      os.system(cmd)
+      total_time_in += time.time()-start
+    print("avg out  time: ", total_time_in/NUM_ITERATIONS)
+
+    total_time_out = 0
+    for x in range(NUM_ITERATIONS):
+      cmd = "export OPT_OUT_TRACKING='true' && python -c 'import time;start= time.time();import mxnet; '"
```

Comment:
Why is there an import time; start=time.time() in the command that is executed, whose result is unused after the command is run?

Reply:
Lifted it from 
https://github.com/aws/deep-learning-containers/blob/ea972e3f40a3cf10e0857e302c24c686f3d59452/test/dlc_tests/container_tests/bin/test_tf_dlc_telemetry_test.py#L55

Reply:
Removing it from this file as time is unused in there..

Reply:
Ah. Understood. @TusharKanekiDey Do you know why this was done?

Reply:
Looks like it was added in redundance. Addressed.

Reply:
Addressed ones in TF & PT here : https://github.com/aws/deep-learning-containers/pull/305